### PR TITLE
Change ES5 getter/setter to ES6

### PR DESCRIPTION
**Pull request by Yuyupo**
IoT.jsCode-DCO-1.0-Signed-off-by: Daniella Barsony bella@inf.u-szeged.hu

### CODE REVIEW

File: src/app/main.js [hunk 1]
```diff
@@ -408,10 +408,10 @@ export default function App() {
           return true;
         }
 
-        const blob = new Blob([session.getFileModelById(session.getActiveID()).getValue()]);
-        FileSaver.saveAs(blob, session.getFileNameById(session.getActiveID()));
-        $('#tab-' + session.getActiveID()).removeClass('unsaved');
-        session.changeFileSavedProperty(session.getActiveID(), true);
+        const blob = new Blob([session.getFileModelById(session.activeID).getValue()]);
+        FileSaver.saveAs(blob, session.getFileNameById(session.activeID));
+        $('#tab-' + session.activeID()).removeClass('unsaved');
```
Comment (rtakacs) [inline src/app/main.js]: Please remove the parentheses after the `activeID`.

File: src/app/main.js [hunk 2]
```diff
@@ -891,16 +891,16 @@ export default function App() {
        */
       $('#command-line-input').keydown((e) => {
         if (e.keyCode === keys.upArrow) {
-          if (session.getCommandCounter() - 1 > -1) {
-            session.setCommandCounter(session.getCommandCounter() - 1);
-            $('#command-line-input').val(session.getCommandList()[session.getCommandCounter()]);
+          if (session.commandCounter - 1 > -1) {
+            session.commandCounter = session.commandCounter - 1;
+            $('#command-line-input').val(session.commandList()[session.commandCounter]);
```
Comment (rtakacs) [inline src/app/main.js]: Unnecessary parentheses after `commandList`.

**Comment by Yuyupo** [comment]
Fixed the typos and moved the getters/setters in one place

**Comment by rtakacs** [comment]
LGTM

**Comment by yichoi** [comment]
LGTM

**Pull request merged**